listenbrainz/webserver/views/stats_api.py

```diff
@@ -13,7 +17,13 @@
                                             UserRecordingStat)
 from data.model.user_release_stat import (UserReleaseRecord,
                                           UserReleaseStat)
-from data.model.user_listening_activity import (UserListeningActivityRecord, UserListeningActivityStat)
+from data.model.user_listening_activity import (
+    UserListeningActivityRecord, UserListeningActivityStat)
+from data.model.user_artist_map import (UserArtistMapRecord,
+                                        UserArtistMapStat,
+                                        UserArtistMapStatJson,
+                                        UserArtistMapStatRange)
+from listenbrainz.config import LISTENBRAINZ_LABS_API_URL
```

Comment:
This leads to race conditions in prod. Could we use `current_app.config['LISTENBRAINZ_LABS_API_URL']` instead?